app/src/main/java/com/amaze/filemanager/filesystem/RootHelper.java

```diff
@@ -44,17 +44,13 @@
      * @return a list of results. Null only if the command passed is a blocking call or no output is
      * there for the command passed
      */
-    public static ArrayList<String> runShellCommand(String cmd) throws ShellNotRunningException {
+    public static List<String> runShellCommand(String cmd) throws ShellNotRunningException {
```

Comment:
> Remember someone going to modify the List will, now, probably copy it (`new ArrayList(runShellCommand())) instead of using it directly, I think not returning an ArrayList works best when you don't want the user to modify the List, for example for private fields in objects.

So it's already here now...

Reply:
I don't understand what you are trying to say.

Reply:
With `runShellCommand()` currently returns `List<String>`, please let me know if there is any changes I need to get a +1.

Reply:
See PR.